ehcache/src/play/cache/EhCacheImpl.java

```diff
@@ -62,10 +63,14 @@ private EhCacheImpl() {
     this.cache = cacheManager.getCache(cacheName, String.class, ValueWrapper.class);
   }
 
-  public static EhCacheImpl getInstance() {
+  public EhCacheImpl instance(@Nonnull Properties playProperties) {
+    if (uniqueInstance == null) {
+      uniqueInstance = new EhCacheImpl();
+    }
```

Comment:
I don't understand this place.
Non-static method in `EhCacheImpl` returns another instance of `EhCacheImpl`?

Reply:
Fixed.